python/example_code/greengrass/snippets/connector_image_classification_usage.py

```diff
@@ -0,0 +1,64 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+"""
+Purpose
+
+Shows how to implement an AWS Lambda function that uses machine learning.
+"""
+
+# snippet-start:[greengrass.python.connector-image-classification-usage.complete]
+import logging
+from threading import Timer
+
+import numpy
+
+import greengrass_machine_learning_sdk as gg_ml
+
+# Assume the inference input image is provided as a local file.
```

Comment:
I don't like "Assume". How about just: "The inference input image." The local file part is obvious.

Reply:
Agreed.